src/components/try-it/call-response/CallResponse.tsx

```diff
@@ -19,8 +28,13 @@ export default function CallResponse() {
                                 <label className="collapse-title text-xl font-medium break-all break-words" htmlFor={`response-${response.timestamp}`}>
                                     {response.url}
                                 </label>
-                                <div className="collapse-content prose max-w-none">
-                                    <pre>{JSON.stringify(response.data, null, 4)}</pre>
+                                <div className="collapse-content prose max-w-none text-l">
+                                    <CopyButton></CopyButton>
+                                    <JsonView value={response.data} style={{...(theme === THEME_DARK ? nordTheme : githubLightTheme), ...{ "font-size": "1rem" }}} indentWidth={25} >
```

Comment:
Why lut the font-size in an object that you then immediately destruct in the same place? Just add it directly.